.changeset/bright-tips-tell.md

```diff
@@ -0,0 +1,14 @@
+---
+"@salt-ds/icons": minor
+---
+
+Add a `color` prop to Icon.
+
+This can be used to specify the color used by Icon and supports: primary, secondary and inherit. The default value is "inherit".
+
+```tsx
+<HomeIcon color="primary" />
+<UserIcon color="secondary" />
+<CloseIcon color="inherit" />
+```
+*Note:* This changes the default icon color from secondary to primary due to it inheriting the default text style. The previous default of secondary was an error.
```

Comment:
I believe when we were doing the theme @pseys said icons should be secondary, has that changed now?

Reply:
I've had this agreed by design yeah